parser/parser.py

```diff
@@ -270,12 +270,17 @@ def process_category_pages(
                 # Process class files
                 href = a.get("href")
                 if href is not None:
-                    file_name = os.path.basename(a["href"].replace(".html", ""))
+                    # Remove any #anchor_id value after the href
+                    href = href.split(".html")[0] + ".html"
```

Comment:
I found that this worked:
> href = href.split("#")[0]

Does the above way of doing it solve some problem?